ddclient.in

```diff
@@ -6728,7 +6728,7 @@ sub nic_porkbun_examples {
 o 'porkbun'
 
 The 'porkbun' protocol is used for porkbun (https://porkbun.com/).
-The API is documented here: https://porkbun.com/api/json/v3/documentation
+The API is documented here: https://api.porkbun.com/api/json/v3/documentation
```

Comment:
I suspect this link should not change; it works today but https://porkbun.com/api/json/v3/documentation is still what's linked from the Porkbun website footer so I'm guessing the documentation will stay on the main website when the API moves off it.

Reply:
I think this documentation will also be moved, it's under the same `/api/json/v3/...` path that is used by the rest of the API.

On both https://porkbun.com/api/json/v3/documentation and https://api.porkbun.com/api/json/v3/documentation, you can see the following notice:

```
## API Hostname
api.porkbun.com

Please note that porkbun.com currently works and has historically been the correct hostname for our API. However, we will be migrating away from that hostname and the API will no function using it at some point in the future.
```

So https://porkbun.com/api/json/v3/documentation will no longer work at some point from my understanding

Reply:
Just double-checked with our dev and can confirm the docs will remain hosted at Porkbun.com, so https://porkbun.com/api/json/v3/documentation will be the correct link for the foreseeable future.

Reply:
They are also hosted on api.porkbun.com. Both links would be correct then. Might as well use the new one imo.

Reply:
Just checked the new link and the css isn't working. Maybe they aren't supporting it yet. Either way doesn't make a different to me. I'll change it back to https://porkbun.com/api/json/v3/documentation

Reply:
Also, just going to mention real quick, I just saw #757 now and I didn't know you worked at porkbun. thanks for chiming in. I thought the api doc would move, but that's not the case.